docs/marketplace/add_to_marketplace/contributing_overview.md

```diff
@@ -14,16 +14,16 @@ Marketplace items can only be created and managed by people with access to the M
 
 Mia-Platform public Marketplace is the place where every Company can find most of the components needed to set up its Project architecture. It is publicly available on Mia-Platform Console, meaning that all Companies can view and make use of its items.
 
-However, a Company might want to have its own internal Marketplace, fulfilled also with components that must not be shared with everyone else outside the Company. 
+However, a Company might want to have its own internal Marketplace, fulfilled also with components that must not be shared with everyone else outside the Company.
 
 To achieve this, a Company Owner or a Project Administrator can create new Company-related components **from the CMS** by specifying their Company's identifier in the `tenantId` field. These items will be visible only to those who are part of that specific Company. This means that, inside the Marketplace of that specific Company, users of that Company will have visibility of both Company-related and public Marketplace components.
 
 Company Owners and Project Administrators can create, delete and update those Markeplace items exclusively available for their Company, but they cannot manage public Marketplace items.  
 The Console Super User role, instead, has permission to manage all Marketplace items from the CMS without any limitations.
 
-If you want to contribute to the **public Mia-Platform Marketplace** with a new component instead, and make it available for everyone, you can [open an issue](https://github.com/mia-platform/community/issues/new?assignees=%40mia-platform%2Fsig-marketplace&labels=marketplace&projects=&template=marketplace-contribution.yaml&title=%5BNew+marketplace+item%5D%3A+) on GitHub, wherein you will find what information is needed for the request.
-
-A Mia-Platform referent will take over the issue and contact you to plan together the addition of the component to the Marketplace on the basis of our guidelines described on this documentation page.
+:::info
+Instead, if you want to contribute to the **public Mia-Platform Marketplace** with a new component, you can visit [this page](/marketplace/add_to_marketplace/marketplace_items_accessibility.md).
```

Comment:
I would not explicitly talk about "public" Marketplace of Mia-Platform in this case. I'd rather focus on the fact that I can make the resources of my Company available to other Companies. This has nothing to do with the Marketplace components supplied by Mia.

Reply:
I replaced "public" anywhere with "accessible/available by/to other Companies" (or something similar)